.eslintrc.json

```
@@ -1,6 +1,17 @@
 {
-  "extends": ["next/core-web-vitals", "prettier"],
+  "extends": [
+    "next/core-web-vitals",
+    "prettier",
+    "plugin:import/recommended",
+    "plugin:import/typescript"
+  ],
   "globals": {
     "JSX": "readonly"
+  },
+  "plugins": ["import"],
+  "rules": {
+    "import/order": ["warn"],
+    "import/no-unused-modules": ["warn"],
+    "import/no-useless-path-segments": ["warn"]
```

- this could be enforced: 
```suggestion
    "import/order": ["error"],
    "import/no-unused-modules": ["error"],
    "import/no-useless-path-segments": ["error"]
```

If we do that, we should automatically correct the code for the user using `eslint --fix` or `prettier`. Can you check that please?